setting-up-multifactor-authentication.md

```diff
@@ -60,3 +60,25 @@ Steps to change your MFA level:
 in the _multifactor authentication_ section, you will see a dropdown menu.
     ![Multifactor section at profile edit page](/images/changing_mfa_step1.png){:class="t-img"}
 2. Select your intended option, enter OTP access code from your authenticator app and click **Update**
+
```

Comment:
Will it be better to add something like ‘You can refer to section “using recovery codes and re-setup a previously enabled MFA”’ at the end of line 42?

Reply:
Yes, great idea. I added the referring link.